### DISCUSSION TITLE
Demo Instance

### DISCUSSION THREAD

**Discussion by meichthys** [ente-io/ente#982] [Enhancements]
Offering a demo instance would make it much easier for people to test out the features without having to self host or sign up for an account and download an app.

**Comment by InvitedToHell** [comment]
I was just looking for one :/
Would indeed be nice

**Comment by 23tux** [comment]
Any news on this? I'd also like to see the features working without having to dump a bunch of photos myself into an account that I may or may not delete afterwards right away

**Comment by pbb72** [comment]
Even a good collection of screenshots would help...

**Comment by timxllr** [comment]
I would really appreciate this too, would help people a lot.

**Comment by flux88** [comment]
100%
